js/batch_ingest.js

```diff
@@ -92,14 +119,53 @@ function openFile(event) {
                 invalidData[invalidCount] = data[j];
                 invalidCount++;
             }
+        }*/
+
+           if(!validateEmail(data, j)) {
+                invalidEmail[invalidEmailCount] = data[j];
+                invalidEmailCount++;
+           }
+           if(!validateID(data, j)) {
+                invalidID[invalidIDCount] = data[j];
+                invalidIDCount++;
+           }
+           if(!validateDate(data, j)) {
+               invalidDate[invalidDateCount] = data[j];
+               invalidDateCount++;
+           }
+           if(!validateCatYear(data, j)) {
+               invalidCataYear[invalidCYCount] = data[j];
+               invalidCYCount++;
+           }
+           if(!validateProg(data, j)) {
+               invalidProg[invalidProgCount] = data[j];
+               invalidProgCount++;
+           }
+           if(!validateDuplicate(data, j)) {
+               dupeID[duplicateIDCount] = data[j];
+               duplicateIDCount++;
+           } else {
```

Comment:
This only actually validates duplicates, because the else is applied to everything that isn't a duplicate. You will have to rework your logic.

Reply:
The invalid stuff still gets passed to the errors page, you just need to make sure that no invalid entries are submitted to the sheet.

Reply:
This seems to be fixed